shell_integration/dolphin/ownclouddolphinpluginhelper.cpp

```diff
@@ -125,17 +124,19 @@ void OwncloudDolphinPluginHelper::slotReadyRead()
                 return;
             }
         } else if (line.startsWith("V2/GET_CLIENT_ICON_RESULT:")) {
-            line.remove(0, QString("V2/GET_CLIENT_ICON_RESULT:").size());
+            line.remove(0, QStringLiteral("V2/GET_CLIENT_ICON_RESULT:").size());
             QJsonParseError error;
             auto json = QJsonDocument::fromJson(line, &error).object();
-            if (error.error != QJsonParseError::NoError)
+            if (error.error != QJsonParseError::NoError) {
                 continue;
+            }
 
             auto jsonArgs = json.value("arguments").toObject();
-            if (jsonArgs.isEmpty())
+            if (jsonArgs.isEmpty()) {
```

Comment:
logging.
The response might contain an error field instead of arguments. pls log the error.